Casks/c/cursor-cli.rb

```diff
@@ -0,0 +1,25 @@
+cask "cursor-cli" do
+  arch arm: "arm64", intel: "x64"
+
+  version "2025.08.08-562252e"
```

Comment:
```suggestion
  version "2025.08.08-f57cb59"
```

It looks like the version is now 2025.08.08-f57cb59.

For what it's worth, if upstream updates the hash multiple times on a given date, we will only catch updates where the hash string is seen as higher than the current cask hash string (as livecheck will treat the cask version as newer than upstream if the upstream hash is lower). Unfortunately we don't have a way of working around this shortcoming at the moment but I figured I'd mention it nonetheless.

Reply:
I think it's fine if we only bump the cask when the date changes. It's basically a built-in throttle.